core/schemas/task.py

```diff
@@ -145,7 +145,8 @@ def _make_request(
             if self.last_run > last_modified:
                 msg = (f"{url}: Last-Modified header ({last_modified_header}) "
                        "before last-run ({self.last_run})")
-                raise RuntimeError(msg)
+                logging.debug(msg)
```

Comment:
By raising a `RuntimeError`, you don't need to worry about checking the response's value in *all* the feeds as you currently do. Besides, this will be reported in the UI as an error, so that users can easily see when a feed is erroring out (without having to look at system logs). Why change this?

Reply:
It's not a error it's just the feed is not updated. There are too many issues about that